radar-upload-frontend/src/components/Patient/PatientTable/Records.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <v-list
+    shaped
+    three-line
+    subheader
+  >
+    <v-layout justify-center>
+      <v-progress-circular
+        class="mt-2"
+        v-show="loading"
+        indeterminate
+        color="primary"
+      />
+      <v-alert
+        type="error"
+        v-text="error"
+        v-show="error"
+        dense
+        text
+      />
+    </v-layout>
+    <v-subheader>
+      <span v-show="!loading">
+        <slot name="fileListSubHeader" />
+      </span>
+      <span v-show="!loading&&patientRecords.length==0">
+        This patient does not have any records
+      </span>
+    </v-subheader>
+    <v-list-group
+      v-for="record in records"
+      :key="record.id"
+      v-model="record.active"
+      no-action
+    >
+      <template #activator>
+        <v-list-item-avatar>
+          <!-- <v-progress-circular
+            v-if="record.status!=='READY'"
+            indeterminate
+            color="primary"
+          /> -->
+          <v-icon
+            color="info"
+            v-if="record.status==='READY'"
+          >
+            mdi-checkbox-marked-circle
+          </v-icon>
+        </v-list-item-avatar>
+
+        <v-list-item-content>
+          <v-list-item-title>
+            Records ID:
+            {{ record.id }} ({{ record.sourceType }})
+          </v-list-item-title>
+          <v-list-item-subtitle>
+            {{ record.message }}
+          </v-list-item-subtitle>
+          <v-list-item-subtitle>
+            {{ record.modifiedDate | localTime }}
+          </v-list-item-subtitle>
+        </v-list-item-content>
+      </template>
+      <!-- view logs dialog -->
+
+      <v-dialog
+        v-model="dialog"
```

Comment:
How does this work? I don't really see a view dialog part in the UI. Where should I expect to see it?

Reply:
it shows the button to view the dialog once the api return the logs. Or else it is hidden

Reply:
ok